offline/packages/PHGarfield/macros/StartScript.sh

```
@@ -0,0 +1,31 @@
+
+bnow=$(awk -v bmin="$bmin" \
+             -v bmax="$bmax" \
+             -v nb="$nb" \
+             -v i="$1" \
+'BEGIN {
+   print bmin + (bmax - bmin) * i / nb
+}')
+
+echo $bnow
+
+output="/direct/phenix+u/workarea/hemmick/code.sphenix/tkh/gas/gasfiles/PART_"$1".gas"
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)